artman/cli/main2.py

```diff
@@ -279,11 +281,14 @@ def normalize_flags(flags, user_config):
     # This allows the user to override the path to api-client-staging or
     # toolkit on his or her machine.
     pipeline_args['local_paths'] = support.parse_local_paths(
-        user_config, flags.input_dir)
+        user_config, flags.root_dir)
 
-    # Save the input directory back to flags if it was not explicitly set.
-    if not flags.input_dir:
-        flags.input_dir = pipeline_args['local_paths']['googleapis']
+    if flags.root_dir:
+        root_dir = flags.root_dir
+    elif pipeline_args['local_paths']['googleapis']:
+        root_dir = pipeline_args['local_paths']['googleapis']
+    else:
+        root_dir = os.getcwd()
```

Comment:
I thought you and @lukesneeringer agreed in offline discussion not to default to the working directory?

Reply:
Yes, we did. This was an accidental revert of my previous change because I switched my working environment in between.